Doc/Case_study.md

```diff
@@ -0,0 +1,263 @@
+# BFI National Archive RAWcooked Case Study  
+**by Joanna White, Knowledge & Collections Developer**  
+  
+At the [BFI National Archive](https://www.bfi.org.uk/bfi-national-archive) we have been encoding DPX sequences to FFV1 Matroska since late 2019. In that time our RAWcooked workflow has evolved with the development of RAWcooked, DPX resolutions and flavours and changes in our encoding project priorities.  Today we have a fairly hands-off automated workflow which handles 2K and 4K image sequences.  This workflow is built on some of the flags developed in RAWcooked by Media Area and written in a mix of Bash shell and Python3 scripts ([BFI Data & Digital Preservation GitHub](https://github.com/bfidatadigipres/dpx_encoding)). In addition to RAWcooked we use other Media Area tools to complete necessary stages of this workflow.  Our encoding processes do not include any alpha channel or audio file processing, but RAWcooked is capable of encoding both into the completed FFV1 Matroska.
+  
+This case study is broken into the following sections:  
+* [Server configurations](#server_config)
+* [Workflow: Image sequence assessment](#assessment)  
+* [Workflow: Encoding the image sequence](#muxing)  
+* [Workflow: Encoding log assessments](#log_assessment)  
+* [Workflow: FFV1 Matroska validation](#ffv1_valid)  
+* [Workflow: FFV1 Matroska decode to image sequence](#ffv1_demux)
+* [Conclusion](#conclusion)
+* [Useful test approaches](#tests)
+* [Additional resources](#links)  
+
+---  
+### <a name="server_config">Server configurations</a>
+  
+To encode our DPX sequences we have a single server that completes this work against 6 different Network Attached Storage (NAS) devices in parallel.  
+  
+Our current server configuration:  
+- Intel(R) Xeon(R) Gold 5218 CPU @ 2.30GHz  
+- 252GB RAM  
+- 32-core with 64 CPU threads  
+- Ubuntu 20.04 LTS  
+- 40Gbps Network card  
+- NAS storage with 40Gbps network card  
+  
+The more CPU threads you have the better your FFmpeg encode to FFV1 will perform. To calculate the CPU threads for your server you can multiply the Threads x Cores x Sockets. So for our configuration this would be 2 (threads) x 16 (sockets) x 2 (cores) = 64. To retrieve these figures we would use Linux's ```lscpu```.
```

Comment:
@digitensions is it possible to have a rough estimate of the encoding speed with one content processed e.g. RGB 10-bit 2K. I am curious to see how the CPU behaves with one single encoding.

Reply:
@digitensions also, if possible, to have an average speed per "parallel" jobs and the count of "parallel" jobs you have, for similar content e.g. all RGB 10-bit 2K, so we can see the difference between a single job and "parallel" usage.